misc/chrome_extension/content-script.js

```diff
@@ -229,20 +231,21 @@ window.onload = function () {
 	 * 
 	 */
 	function sendMouseElement() {
-		ableToClick = true;
-		ableToMouseOver = false;
+		__ableToClick = true;
+		__ableToMouseOver = false;
 
-		let jsonToSend = {}
-		jsonToSend['mouse'] = getXPath(prev);
+		let jsonToSend = {
+			mouse: getXPath(__previousSelecetedElement)
+		};
 
 		try {
-			if (prev) {
-				prev.className = prev.className.replace(/\bhighlight\b/, '');
-				prev = undefined;
+			if (__previousSelecetedElement) {
+				__previousSelecetedElement.className = __previousSelecetedElement.className.replace(/\bhighlight\b/, '');
```

Comment:
Some places use `__highlight`, others `highlight`. 
Is it an issue or two different things?